### DISCUSSION TITLE
Drill to Detail Data

### DISCUSSION THREAD

**Discussion by alberk8** [apache/superset#27460] [Q&A / Help]
How do I export the Drill to Detail Data from the Dashboard.

![image](https://github.com/apache/superset/assets/12510130/e14813a8-4d04-4afd-b6e0-b4a4025e541f)

**Comment by MikeDel001** [comment]
I have also been looking to do this, thought i was missing something but perhaps it is not a feature that is available yet? if i'm wrong i would be forever grateful if someone could advise how to export the drill to detail data.
